components/StyledText.js

```diff
@@ -2,5 +2,5 @@ import * as React from 'react';
 import { Text } from 'react-native';
 
 export function MonoText(props) {
-  return <Text {...props} style={[props.style, { fontFamily: 'Inter-Black' }]} />;
+  return <Text {...props} style={[props.style]} />;
```

Comment:
what is the purpose of `MonoText`?

Reply:
at the moment it has no purpose. Before it was use for applying the same fontFamily and style flexibility to all the text components used across the app. Correct ? 
I had to delete the fontFamily cause was creating issues. 
Hopefully I will give a purpose to this component once again when I'm done with the current exercise.